### PR TITLE
Update it.json

### PR DESCRIPTION
**Pull request by masterix21**

### CODE REVIEW

File: json/it.json [hunk 1]
```diff
@@ -247,60 +247,60 @@
...
     "Jersey": "Jersey",
-    "Jordan": "Jordan",
-    "July": "July",
-    "June": "June",
-    "Kazakhstan": "Kazakhstan",
+    "Jordan": "Giordania",
+    "July": "Luglio",
+    "June": "Giugno",
+    "Kazakhstan": "Kazakistan",
     "Kenya": "Kenya",
```
Comment (caouecs) [inline json/it.json:268]: In exclude file ?

**Review by caouecs — CHANGES_REQUESTED** [review]
Hello,
just a question on exclude tag

File: json/it.json (hunk 1 above)
Comment (masterix21) [reply json/it.json:268]: Many countries have no translations. English "Kenya" is the same in Italian.

Comment (masterix21) [reply json/it.json:268]: If you like, I can add these exclusions to `script/excludes/it.php`.

Comment (caouecs) [reply json/it.json:268]: yes ;)

Comment (masterix21) [reply json/it.json:268]: Done ;)

File: script/excludes/it.php [hunk 2]
```diff
@@ -11,9 +11,135 @@
...
+    'ID',
     'Email',
     'Oh no',
     'Editor',
     'Password',
     'Dashboard',
+    'No',
+
+    /**
+     * Countries
```
Comment (andrey-helldar) [inline script/excludes/it.php:23]: The country names do not need to be specified in the exclusion list, which is needed to generate the todo.md file.

The list of countries is already listed in the global list for all localizations: [script/excludes/_all.php](https://github.com/Laravel-Lang/lang/blob/master/script/excludes/_all.php)

**Review by andrey-helldar — CHANGES_REQUESTED** [review]

File: script/excludes/it.php (hunk 2 above)
Comment (caouecs) [reply script/excludes/it.php:23]: ok

**Comment by caouecs** [comment]
Grazie

**Pull request merged**